### PR TITLE
javascript-loops-practice | Paul Park

### PR DESCRIPTION
**Pull request by paulpark330**
javascript-loops-practice

### CODE REVIEW

File: javascript-loops-practice/filter-out-strings.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var newArr = [];
+  for (let i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```
Comment (codingcodymiller) [inline javascript-loops-practice/filter-out-strings.js:5]: This isn't filtering out strings, it's keeping numbers. What if there were other values in the array such as booleans or objects?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! One note for you:

File: javascript-loops-practice/filter-out-strings.js (hunk 1 above)
Comment (paulpark330) [reply javascript-loops-practice/filter-out-strings.js]: Hey Cody, Thanks for the feedback! 

I commented some pseudocode in the filter-out-strings.js 
Let me know what you think!

File: javascript-loops-practice/filter-out-strings.js [hunk 2]
```diff
@@ -2,9 +2,19 @@
 function filterOutStrings(values) {
   var newArr = [];
   for (let i = 0; i < values.length; i++) {
-    if (typeof values[i] === 'number') {
+    if (typeof values[i] === 'string') {
+      continue;
+    } else {
```
Comment (codingcodymiller) [inline javascript-loops-practice/filter-out-strings.js:7]: Nice!! The pseudocode looks great and matches your function perfectly 🤓 👌 

However, the `continue` keyword is often an indicator that you have your logic backwards. If your work for a condition is to "move on", odds are you should really be looking for the opposite. This function will be more straightforward If your condition was `typeof values[i] !== 'string'`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-loops-practice/filter-out-strings.js (hunk 2 above)
Comment (paulpark330) [reply javascript-loops-practice/filter-out-strings.js:7]: Ohhh that makes perfect sense. Got it!!

Comment (paulpark330) [reply javascript-loops-practice/filter-out-strings.js:7]: Is it okay to leave a code block completely empty?

**Review by paulpark330 — COMMENTED** [review]
Oops what did I do here 😅
Must have clicked on Review..

File: javascript-loops-practice/filter-out-strings.js [hunk 3]
```diff
@@ -3,7 +3,6 @@ function filterOutStrings(values) {
   var newArr = [];
   for (let i = 0; i < values.length; i++) {
     if (typeof values[i] === 'string') {
-      continue;
     } else {
       newArr.push(values[i]);
     }
```
Comment (codingcodymiller) [inline javascript-loops-practice/filter-out-strings.js]: Here is what you want to do:
```
for (var i = 0; i < values.length; i++) {
    if (typeof values[i] !== 'string') {
      newArr.push(values[i]);
    }
}
```

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-loops-practice/filter-out-strings.js (hunk 3 above)
Comment (paulpark330) [reply javascript-loops-practice/filter-out-strings.js]: I see how that works!
I didn't think about changing my pseudocode.
Thank you, learned a lot!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**